store/android/src/main/java/com/jakewharton/sdksearch/store/DbCallback.kt

```diff
@@ -3,15 +3,15 @@ package com.jakewharton.sdksearch.store
 import android.arch.persistence.db.SupportSQLiteDatabase
 import android.arch.persistence.db.SupportSQLiteOpenHelper
 
-private const val VERSION = 2
+private const val VERSION = 3
 
 internal class DbCallback : SupportSQLiteOpenHelper.Callback(VERSION) {
   override fun onCreate(db: SupportSQLiteDatabase) {
     db.execSQL(ItemModel.CREATE_TABLE)
   }
 
   override fun onUpgrade(db: SupportSQLiteDatabase, oldVersion: Int, newVersion: Int) {
-    if (oldVersion < 2) {
+    if (oldVersion < 3) {
```

Comment:
Why not sure `VERSION` here directly?

Reply:
Because it's not guaranteed future migrations will want to delete the table.

Reply:
I see it is in the `onUpgrade` call now. Makes sense. Thanks.